### PR TITLE
Update Metadata IG description

### PR DESCRIPTION
**Pull request by riedde**
Updating text and co-chairs.

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]
Thanks for updating this.

File: community/interest-groups.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ The aim of the Interest Group is to discuss the use of MEI for the encoding of m
...
+**Administrative Co-Chairs: [Sophia Dörner](mailto:sophia.doerner@hu-berlin.de) and [Dennis Ried](mailto:dennis.ried@uni-paderborn.de)**.<br/>
+The aim of the Metadata and Cataloging Interest Group is to discuss the use of MEI for the encoding of musical metadata; to make recommendations regarding the development of metadata specific elements; to promote the sharing of and collaboration on tools for projects concerned with MEI metadata such as cataloging projects. 'Cataloging' in this context may include a wide range of projects such as thematic catalogs ('Werkverzeichnisse'), library catalogs, and other descriptions of musical works or sources. The group is open to all members of the MEI community. Members enroll simply by subscribing to the group's mailing list: [https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig](https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig).
```
Comment (musicEnfanthen) [inline community/interest-groups.md:27]:
```suggestion
The aim of the Metadata and Cataloging Interest Group is to discuss the use of MEI for the encoding of musical metadata; to make recommendations regarding the development of metadata-specific elements; to promote the sharing of and collaboration on tools for projects concerned with MEI metadata such as cataloging projects. 'Cataloging' in this context may include a wide range of projects such as thematic catalogs ('Werkverzeichnisse'), library catalogs, and other descriptions of musical works or sources. The group is open to all members of the MEI community. Members enroll simply by subscribing to the group's mailing list: [https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig](https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig).
```

**Review by musicEnfanthen — COMMENTED** [review]
Too late to the party, but found just a small typo? Otherwise looks fine to me, too.

File: community/interest-groups.md (hunk 1 above)
Comment (rettinghaus) [reply community/interest-groups.md:27]: @musicEnfanthen Seems like I can't commit your change. Could you please open a separate PR?

Comment (musicEnfanthen) [reply community/interest-groups.md:27]: @riedde Dou you mind to create another PR to update the typo? Sorry for spotting it only after the merge.

**Pull request merged**